examples/apps/aml/standalone-cipher/worker.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2019 Banco Santander S.A.
```

Comment:
Is it possible to reuse the existing file https://github.com/hyperledger/avalon/blob/master/examples/common/python/worker/worker_details.py

Reply:
Yes, you are right in both you comments probably most of the functionality can be reused. Is just duplicated because we were using it completely decoupled from Avalon. I'm gonna reuse them and open a new PR on that.